specs2/BUILD

```diff
@@ -1,13 +1,14 @@
+load("//scala:scala_import.bzl", "scala_import")
+
 package(default_visibility = ["//visibility:public"])
-java_import(
+scala_import(
```

Comment:
Why is this change needed?

Reply:
I think it's a leftover from our change with the jars, I'll undo and see what happens.

Reply:
Ok, I reverted it back to java_import, still works!